database/src/main/resources/db/migrations/gendox-core/V20231213_154530__Add_cohere_model.sql

```diff
@@ -0,0 +1,9 @@
+INSERT into gendox_core.ai_models
+(model, url, name, price, created_at, updated_at, description)
+select 'command', 'https://api.cohere.ai/v1/generate', 'command', 0.02, now(), now(), 'command text generation endpoint'
+where not exists(SELECT * FROM gendox_core.ai_models where model = 'command');
```

Comment:
then `name` column should be handled as a **static unique identifier**.
This should be in the select statement, and the name should be unique across all models.

So, it should be named like `COHERE_COMMAND`, the 'COHERE' is important here, since multiple model could have the model_name as `command`

```sql
INSERT into gendox_core.ai_models
(model, url, name, price, created_at, updated_at, description)
select 'command', 'https://api.cohere.ai/v1/generate', 'COHERE_COMMAND', 0.02, now(), now(), 'command text generation endpoint'
where not exists(SELECT * FROM gendox_core.ai_models where name = 'COHERE_COMMAND');
```